jOOQ/src/main/java/org/jooq/impl/DefaultDSLContext.java

```diff
@@ -2817,6 +2818,26 @@ public Queries ddl(Table<?> table, DDLFlag... flags) {
         return new DDL(this, flags).queries(table);
     }
 
+    @Override
+    public Queries ddl(Table[] tables) {
```

Comment:
In order to improve cohesion, the existing methods will call the new methods rather than passing responsibility directly to `DDL`